### DISCUSSION TITLE
Website: "dependancies"

### DISCUSSION THREAD

**Discussion by cweiske** [alpinejs/alpine#3205] [7. Bugs (archived)]
The website https://alpinejs.dev/ says:
>  x-effect
> Execute a script each time one of its dependancies change

"dependancies" is wrong, the "a" has to be replaced with an "e": dependencies

**Comment by ekwoka** [comment]
Looks like an easy PR for you to take care of!

Congrats 🎉

**Reply by cweiske** [reply]
Then please tell me where the source code for the website is.

**Reply by ekwoka** [reply]
It's in this same repo you're commenting on: https://github.com/alpinejs/alpine/tree/main/packages/docs

**Reply by cweiske** [reply]
You linked to the documentation, but that does not contain the index page that is shown at https://alpinejs.dev/

**Reply by sobstel** [reply]
I've just followed same path today. I've noticed the typo, wanted to post a simple PR, but there doesn't seem to be index page on the repo.

**Reply by ekwoka** [reply]
Yes, the actual docs repo is a different private repo.

**Reply by benallard** [reply]
Just doing my part and reporting that the typo is still there.
